dsdl_compiler/libcanard_dsdl_compiler/code_type_template.tmpl

```diff
@@ -0,0 +1,412 @@
+/*
+ * UAVCAN data structure definition for libcanard.
+ *
+ * Autogenerated, do not edit.
+ *
+ * Source file: ${t.source_file}
+ */
+%if not t.header_only
+#include "${t.header_filename}"
+#include "canard.h"
+%endif
+
+#ifndef _UAVCAN_INTERNAL_SATURATE
+#define _UAVCAN_INTERNAL_SATURATE(x, max) ( ((x) > max) ? max : ( (-(x) > max) ? (-max) : (x) ) );
+#endif
```

Comment:
Names starting with an underscore followed by a capital letter are reserved for the implementation. I suggest using a different prefix, e.g. `CANARD_`